Assets/MixedRealityToolkit.Services/InputSystem/InputSystemGlobalListener.cs

```diff
@@ -15,18 +15,28 @@ public class InputSystemGlobalListener : MonoBehaviour
     {
         private bool lateInitialize = true;
 
+        private IMixedRealityInputSystem inputSystem = null;
+
         /// <summary>
-        /// Prefer using EnsureInputSystemValid(), which will only await if the input system isn't
-        /// yet valid.
+        /// The active instance of the input system.
         /// </summary>
-        [Obsolete("Prefer using EnsureInputSystemValid()")]
-        protected readonly WaitUntil WaitUntilInputSystemValid = new WaitUntil(() => MixedRealityToolkit.InputSystem != null);
```

Comment:
This is technically a breaking change, and also a break that doesn't accrue any value to other developers.

Reply:
i can put something back. the new pattern does not work with a readonly WaitUntil instance. also, it was used in exactly one place (that has been updated)

Reply:
Just confirmed. There are no references in the MRTK codebase other than the call within EnsureInputSystemValid().

Not sure if there is a clean path to keep this... We can discuss

Reply:
We did a cost/benefit analysis and feel this change is the correct thing to do.